### DISCUSSION TITLE
Problem with the server?

### DISCUSSION THREAD

**Discussion by ghost** [rubenarslan/formr.org#512] [Q&A]
Hello!

Suddenly the following error message is shown although I have not altered the R code and everything worked out smoothly so far:
<img width="1158" alt="Screenshot 2024-01-18 at 09 20 33" src="https://github.com/rubenarslan/formr.org/assets/157002957/e535f007-68d6-4648-a838-91a150f5424e">

<img width="1138" alt="Screenshot 2024-01-18 at 09 33 57" src="https://github.com/rubenarslan/formr.org/assets/157002957/58f3ed0c-5800-499a-8e9a-4ca49431e9db">

Is there a problem with the server?

Best regards,
Lu

**Comment by floKuehl** [comment]
Hey Lu, I experience the same problem since 8.45/9 am CET.

**Comment by cyriltata** [comment]
The opencpu server was temporarily not responding. Please can you try again?

best
Cyril

**Reply by ghost** [reply]
It's working again, thank you for your help!

**Comment by jkelgs** [comment]
Hello, 
The opencpu server seems to be down again since more than an hour:
<img width="706" alt="Bildschirmfoto 2024-01-24 um 14 02 40" src="https://github.com/rubenarslan/formr.org/assets/114669158/57481a97-bf57-415d-ae6d-4419f759a260">

Best,
Fabian

**Comment by jkelgs** [comment]
Hello, 
The opencpu server seems to be down again since more than an hour:
<img width="706" alt="Bildschirmfoto 2024-01-24 um 14 02 40" src="https://github.com/rubenarslan/formr.org/assets/114669158/57481a97-bf57-415d-ae6d-4419f759a260">

Best,
Fabian

**Reply by jkelgs** [reply]
it's now working again!

**Comment by xpressio** [comment]
Dear formr team,

also seeing the above error message here.

Thanks for any help.

Best,
Max

**Comment by jkelgs** [comment]
Dear all,

OpenCPU seems to be down since about an hour....

Best,
Fabian

**Reply by ulyngs** [reply]
Same here!

**Reply by xpressio** [reply]
Back up again.
